yetanotherco/aligned_layer
# fix(explorer): show correct cost per proof and remove inspects

> [!IMPORTANT]  
> We should test in stage first and compare results. 
> This also needs a redeploy of the contracts. 
> This PR could have conflicts w/ #843.

# Changes

* ~~merge before #651~~
* add emit to payment service contract with merkle root as index and emit fee directly
* read event from explorer
* update gas_per_proof to use fee directly instead of old hardcoded value multiplied by current gas fetched
* add migrate with name change from cost to fee
* remove SQL logs and close #857 

# Test

* Start aligned locally
* Run explorer and verify everything works, prices should now be higher than the deployed version
